bcda/client/bluebutton_test.go

```diff
@@ -949,12 +952,16 @@ func hasDefaultRequestHeaders(t *testing.T, req *http.Request) {
 	assert.NotEmpty(t, req.Header.Get(constants.BBHeaderOriginQ))
 	assert.NotEmpty(t, req.Header.Get(constants.BBHeaderOriginQC))
 }
-func hasDefaultRequestHeadersNoQuery(t *testing.T, req *http.Request) {
-	assert.NotEmpty(t, req.Header.Get(constants.BBHeaderTS))
-	assert.NotEmpty(t, req.Header.Get(constants.BBHeaderOriginURL))
-	assert.NotEmpty(t, req.Header.Get(constants.BBHeaderOriginQID))
-	assert.Empty(t, req.Header.Get(constants.BBHeaderOriginQ))
-	assert.NotEmpty(t, req.Header.Get(constants.BBHeaderOriginQC))
+func hasContentTypeURLEncodedHeader(t *testing.T, req *http.Request) {
```

Comment:
The verification that I removed was one that I had added previously for this same ticket and is no longer needed.